finance/proposal-calls/cycle3/user-dev-support-rse.md

```diff
@@ -0,0 +1,87 @@
+### Title
+
+Research Software Engineer: User and Developer Support
+
+### Project Team
+
+- Adrian Price-Whelan
```

Comment:
Just to be clear: I put my name down to help as manager of this role, not for the RSE role itself!
```suggestion
- Adrian Price-Whelan (manager)
```